bokun.yaml

```diff
@@ -5952,14 +6271,62 @@ definitions:
         description: 'Only used if hasOpeningHours == true. The default opening hours by weekday. These may be overridden by seasonal opening hours.'
       seasonalOpeningHours:
         type: array
-        description: 'Optional seasonal opening hours. If date falls within the season then this will override the default opening hours.'
+        description: 'Deprecated. Optional seasonal opening hours. If date falls within the season then this will override the default opening hours.'
         items:
           $ref: '#/definitions/SeasonalOpeningHours'
+      displaySettings:
+        type: object
+        properties:
+          showPickupPlaces:
+            type: boolean
+            description: 'Specifies if pick up places are shown.'
+          showRouteMap:
+            type: boolean
+          selectRateBasedOnStartTime:
+            type: boolean
+          customFields:
+            type: array
+            items:
+              type: object
+            description: 'Deprecated.'
+      hasBoxes:
+        type: boolean
+        description: 'Not working.'
```

Comment:
I would rather suggest leaving this empty and/or filling a bug.

Reply:
Will review with Tadas and file a bug if needed.

Reply:
bug filed